portkey_ai/api_resources/global_constants.py

```diff
@@ -33,3 +33,4 @@
 PORTKEY_GATEWAY_URL = PORTKEY_BASE_URL
 PORTKEY_API_KEY_ENV = "PORTKEY_API_KEY"
 PORTKEY_PROXY_ENV = "PORTKEY_PROXY"
+OPEN_AI_API_KEY = "DUMMY-KEY"
```

Comment:
I think this is not required anymore, right? We can remove if thats the case.

Reply:
We need some value as a placeholder. Even it has no significance, but we need some string to pass.